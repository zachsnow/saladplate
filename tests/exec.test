Evaluate a simple expression: $(( echo "Hello, world!" ))
