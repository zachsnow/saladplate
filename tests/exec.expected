Evaluate a simple expression: Hello, world!
